Bio/AlignIO/MafIO.py

```diff
@@ -123,7 +121,6 @@ def write_alignment(self, alignment):
         Writes every SeqRecord in a MultipleSeqAlignment object to its own
         MAF block (beginning with an 'a' line, containing 's' lines)
         """
-
```

Comment:
That was a deliberate change to follow PEP257 docstring style guidelines as now enforced in TravisCI with pydocstyle - see https://github.com/biopython/biopython/commit/49e7da7b44c913143956e3662f4ada36937a3dc8